### PR TITLE
docs(path): add more details to `ParsedPath` and other examples

### PR DESCRIPTION
**Pull request by ooker777**

### CODE REVIEW

File: path/_interface.ts [hunk 1]
```diff
@@ -3,14 +3,22 @@
 
 /**
  * A parsed path object generated by path.parse() or consumed by path.format().
+ * parse("c:\\path\\dir\\index.html")
```
Comment (kt3k) [inline path/_interface.ts:6]: Can you use code block and `@example` directive here?

**Review by kt3k — COMMENTED** [review]
Thanks for the PR. Left one comment

File: path/_interface.ts (hunk 1 above)
Comment (ooker777) [reply path/_interface.ts:6]: Thanks for the feedback. Is there any guideline or convention about this? The contributing guideline doesn't talk about this

https://github.com/denoland/deno_std/blob/main/.github/CONTRIBUTING.md

**Review by kt3k — APPROVED** [review]
LGTM

**Review by iuioiua — APPROVED** [review]
I've made a few formatting nits to make it a little more consistent with the rest of the codebase. Either way, great work! Thank you for your contribution. LGTM.

**Pull request merged**